WorkflowAdminBundle/DependencyInjection/OpenOrchestraWorkflowAdminExtension.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace OpenOrchestra\WorkflowAdminBundle\DependencyInjection;
+
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\Config\FileLocator;
+use Symfony\Component\HttpKernel\DependencyInjection\Extension;
+use Symfony\Component\DependencyInjection\Loader;
+
+/**
+ * This is the class that loads and manages your bundle configuration
+ *
+ * To learn more see {@link http://symfony.com/doc/current/cookbook/bundles/extension.html}
+ */
+class OpenOrchestraWorkflowAdminExtension extends Extension
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function load(array $configs, ContainerBuilder $container)
+    {
+        $loader = new Loader\YamlFileLoader($container, new FileLocator(__DIR__.'/../Resources/config'));
+        $loader->load('form.yml');
+        $loader->load('leftpanel.yml');
```

Comment:
can we supress the term leftPanel and replace it with navigationPanel ? I find this naming innacurate
Same as rightPanel which should mentionned as blockPanel on my mind